fixtures/mega-lite/washbot-led-cymk-300.json

```diff
@@ -0,0 +1,604 @@
+          "type": "Color",
+          "name": "Lime Green",
+          "colors": ["#70ff00"]
+        },
+        {
+          "type": "Color",
+          "name": "Orange",
+          "colors": ["#ff7000"]
+        },
+        {
+          "type": "Color",
+          "name": "Green",
+          "colors": ["#00ff00"]
+        },
+        {
+          "type": "Color",
+          "name": "Magenta",
+          "colors": ["#ff00ff"]
+        }
+      ]
+    },
+    "Color Section 3": {
+      "slots": [
+        {
+          "type": "Open"
+        },
+        {
+          "type": "Color",
+          "name": "Light Pink",
+          "colors": ["#ff40aa"]
+        },
+        {
+          "type": "Color",
+          "name": "Teal",
+          "colors": ["#00ffff"]
+        },
+        {
+          "type": "Color",
+          "name": "RBG",
+          "colors": ["#ff0000", "#0000ff", "#00ff00"]
+        },
+        {
+          "type": "Color",
+          "name": "Yellow",
+          "colors": ["#ffff00"]
+        }
+      ]
+    },
+    "Color Section 4": {
+      "slots": [
+        {
+          "type": "Open"
+        },
+        {
+          "type": "Color",
+          "name": "4789K",
+          "colors": ["#ffffc6"]
+        },
+        {
+          "type": "Color",
+          "name": "3513K",
+          "colors": ["#ffc489"]
+        },
+        {
+          "type": "Color",
+          "name": "3301K",
+          "colors": ["#ffbe7e"]
+        },
+        {
+          "type": "Color",
+          "name": "5262K",
+          "colors": ["#ffe9d9"]
+        }
+      ]
+    }
+  },
+  "availableChannels": {
+    "Color Wheel": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 1],
+          "type": "WheelSlot",
+          "slotNumber": 1
+        },
+        {
+          "dmxRange": [2, 17],
+          "type": "WheelSlot",
+          "slotNumber": 2
+        },
+        {
+          "dmxRange": [18, 32],
+          "type": "WheelSlot",
+          "slotNumber": 3
+        },
+        {
+          "dmxRange": [33, 46],
+          "type": "WheelSlot",
+          "slotNumber": 4
+        },
+        {
+          "dmxRange": [47, 64],
+          "type": "WheelSlot",
+          "slotNumber": 5
+        },
+        {
+          "dmxRange": [65, 79],
+          "type": "WheelSlot",
+          "slotNumber": 6
+        },
+        {
+          "dmxRange": [80, 97],
+          "type": "WheelSlot",
+          "slotNumber": 7
+        },
+        {
+          "dmxRange": [98, 125],
+          "type": "WheelSlot",
+          "slotNumber": 8
+        },
+        {
+          "dmxRange": [126, 190],
+          "type": "WheelRotation",
+          "speedStart": "fast CW",
+          "speedEnd": "slow CW"
+        },
+        {
+          "dmxRange": [191, 255],
+          "type": "WheelRotation",
+          "speedStart": "slow CCW",
+          "speedEnd": "fast CCW"
+        }
+      ]
+    },
+    "Color Section 1": {
```

Comment:
These channels (as well as the wheels) are called "Color Selection" in the manual, not "Color Section".

Reply:
Good catch! Guess I read that one a little to quickly